src/Vezeeta.Infrastructure/Data/AppDbContext.cs

```diff
@@ -22,5 +25,96 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
         base.OnModelCreating(modelBuilder);
         modelBuilder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());
+
+        SpecializationsDataSeeding(modelBuilder);
+        RolesDataSeeding(modelBuilder);
+    }
+
+    private void SpecializationsDataSeeding(ModelBuilder builder)
+    {
+
+        builder.Entity<Specialization>()
+       .HasData(new List<Specialization>
+          {
+                    new Specialization
+                    {   ID = 1,
+                        SpecializationName="Dermatology"
+                    },
+                    new Specialization
+                    {
+                        ID = 2,
+                        SpecializationName="Dentistry"
+                    },
```

Comment:
The `OnModelCreating()` method has been abused for a long time by Devs who add all Model Configs in this method. Imagine a large system which has too many configs, this method would be 300 Lines long and try editing it 🥱. That's why EF has support for Config classes that can separate each model's config. [A reference can be viewed here](https://github.com/YoussefWaelMohamedLotfy/BookingClone/blob/main/src/Services/BookingClone.Infrastructure/EntityConfigurations/AttractionConfiguration.cs). 

FYI, Line 27 will register these config classes automatically 👍🏻